package.json

```diff
@@ -42,28 +55,48 @@
     "single-spa": ">=4.0.0"
   },
   "devDependencies": {
+    "@angular-builders/custom-webpack": "~9.1.0",
     "@angular-devkit/architect": "~0.901.0",
     "@angular-devkit/build-angular": "~0.901.0",
-    "@angular-devkit/core": "^9.1.0",
+    "@angular-devkit/core": "~9.1.0",
     "@angular-devkit/schematics": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/compiler-cli": "^9.1.0",
-    "@angular/core": "^9.1.0",
-    "@schematics/angular": "^9.1.0",
+    "@angular/animations": "~9.1.0",
+    "@angular/cli": "~9.1.0",
+    "@angular/common": "~9.1.0",
+    "@angular/compiler": "~9.1.0",
+    "@angular/compiler-cli": "~9.1.0",
+    "@angular/core": "~9.1.0",
+    "@angular/forms": "~9.1.0",
+    "@angular/platform-browser": "~9.1.0",
+    "@angular/platform-browser-dynamic": "~9.1.0",
+    "@angular/router": "~9.1.0",
+    "@schematics/angular": "~9.1.0",
     "@types/jest": "^24.0.11",
     "@types/node": "^11.11.3",
     "@types/webpack-merge": "^4.1.3",
     "@typescript-eslint/eslint-plugin": "^1.4.2",
     "@typescript-eslint/parser": "^1.4.2",
     "cpx": "^1.5.0",
+    "cypress": "^4.3.0",
     "eslint": "^5.15.2",
     "jest": "^24.5.0",
     "rimraf": "^2.6.2",
+    "rxjs": "~6.5.5",
+    "start-server-and-test": "^1.10.11",
     "ts-jest": "^24.0.0",
+    "tslib": "^1.11.1",
     "typescript": "3.7.5",
-    "webpack": "^4.29.6"
+    "webpack": "^4.29.6",
+    "webpack-cli": "^3.3.11",
+    "zone.js": "~0.10.3"
   },
   "dependencies": {
     "single-spa": ">= 5.1.1"
+  },
+  "prettier": {
```

Comment:
Why is this here, when single-spa-angular doesn't currently use prettier to autoformat files? I am open to adding prettier to the project, but that would necessitate installing prettier, pretty-quick, and husky as devDependencies, and then adding a pre-commit git hook. I'd rather not have a prettier config unless we're actually using prettier.

Reply:
True. That is what I'm planning to do next.

I actually have an auto-formatting in my vscode and it relies on the project config. We could leave that config here and I will add prettier/husky/etc. in the next PR. Thoughts?

Reply:
Sure, sounds good